### ISSUE TITLE
Add Name of Current Participant to Screen

### ISSUE THREAD

**Issue by WillHaack** [ByunLab/staRt-ionic#367]
An SLP may forget to switch the active participant when doing a new session. To prevent this, the Quests/Quizzes should display the name/initials of the participant (possibly in the top center?) on the screen. 

@helencarey

**Comment by helencarey** [comment]
We should probably discuss with @tarabyun if there will be any changes to the Clinician > Participant relationship in preparation for the appStore release. (Currently we can't have a participant w/o a clinician. Will this still be the case going forward?)
 
TODO:
- [ ] move code the root template so it is persistent throughout the app (generally this is the ux 'best practice')
- [ ] make the participant label conditional. (If there is no participant label is on screen, it can serve as a reminder that no active profile is selected.)

**Comment by tarabyun** [comment]
I don't have an immediate plan to give end users access without clinicians. Also, relative to this particular feature--even if we have someone using the app without a clinician, wouldn't this just display their login email and then their profile username? So it wouldn't be redundant even if it were just a solo user.

**Comment by helencarey** [comment]
@tarabyun Cool - that makes our lives easier. 
We switched to displaying the 'pretty name' from the firebase account, so currently it looks a little redundant:
![Simulator Screen Shot - iPad Air 2 - 2019-09-03 at 11 26 33](https://user-images.githubusercontent.com/8650944/64188023-9b94ea00-ce3f-11e9-9019-a8abbc8bebd4.png)

We could return to displaying the clinician email. 

Or we could just omit the clinician identifier altogether if: 
- staRt is not likely to be used in setting where more than 1 clinician is using a school-owned iPad, 
- it is not likely that a clinician would want multiple clinician accts. (ex. an SLP covers 2 schools. She registers student profiles for school#1 with her school#1 email/firebase acct, and students for school#2 with her school#2 email/firebase acct.)

**Comment by tarabyun** [comment]
I think both of the scenarios you listed are pretty low-likelihood. To my knowledge, we haven't had any cases of multiple clinician users sharing an iPad, and I'm almost certain we don't have anyone who is actively using the app with multiple accounts. So I think omitting the clinician identifier should be fine.

**Comment by helencarey** [comment]
**QA Note**
**changes affect:** Quiz and Quest activity screens

Adds name on Active Profile to screen for Quiz and Quest activities, so a user doesn’t accidentally complete an activity (write data) to the wrong account.

direct testing:
- [ ] participant name appears on upper left corner of the screen whenever Quiz or Quest activities launch 

in general:
- [ ] text is legible
- [ ] text is meaningful (in other words, this addition of info on the screen is helpful, not confusing or distracting )


**minor issue** 
Longer user names are likely to wrap and might look funky.
In the future, we might want to write rules to handle long user names here and in other places, like on several of the Profiles screens. To do this we would either grab the text before a the 1st space (and assume its a first name), or make the font smaller for names with more than x# characters.


**short username**
![Simulator Screen Shot - iPad Air 2 - 2019-09-16 at 11 06 22](https://user-images.githubusercontent.com/8650944/64985295-8df55080-d892-11e9-982b-0f3f8028fdc4.png)

**long username**
![Simulator Screen Shot - iPad Air 2 - 2019-09-16 at 11 52 19](https://user-images.githubusercontent.com/8650944/64985323-9baad600-d892-11e9-9ee0-f8b5ec36fe66.png)

**Comment by helencarey** [comment]
@helencarey the name is there, but the styles are strange b/c I did them prior to #378.
Also the name will appear in Free Play also due to #378.

@Twylah, please do the QA note for feedback. 
After that I'll create a new issue to:
- fix the styles for on-protocol signposting
- update styles for name label on off-protocol sessions
- adjust size for long names everywhere

**Comment by helencarey** [comment]
Also, if there is no profile selected and you go to Free Play (this can happen if you log in and go to Free Play immediately), then the starfish and never shows up.  

@Tara If someone tries this, should we tell them to select a profile first, or should we just give them a default starfish? For example, if there is no active profile, I believe the lpc-order defaults to 35. Is there a good default starfish value?

**Comment by tarabyun** [comment]
Wow, sorry I dropped the ball on this for a month :( I think we just ran into this issue with a user who was confused about how to view the white line. We could set the default to whatever value is saved for a 10-year-old female. But I do think we should also provide a pop-up warning that this is a default value and that users should (a) complete the tutorial to learn how to use the wave, (b) create a profile in order to access a customized target. We can discuss this today...
